src/fugw/mappings/barycenter.py

```diff
@@ -270,7 +277,7 @@ def fit(
         duals = None
         losses_each_bar_step = []
 
-        for _ in range(nits_barycenter):
+        for idx in range(nits_barycenter):
```

Comment:
You're not using `idx`, are you ?  If not, you should not introduce it.

Reply:
I mainly use it with callbacks and I intend to use it in a future PR for improved logging when using barycenters.